climada/hazard/test/test_tc_tracks.py

```diff
@@ -376,13 +426,13 @@ def test_get_extent(self):
         """Test extent/bounds attributes."""
         storms = ['1988169N14259', '2002073S16161', '2002143S07157']
         tc_track = tc.TCTracks()
-        tc_track.read_ibtracs_netcdf(storm_id=storms)
-        bounds = (153.4752, -23.2000, 258.7132, 17.5166)
+        tc_track.read_ibtracs_netcdf(storm_id=storms, provider=["usa", "bom"])
+        bounds = (153.585022, -23.200001, 258.714996, 17.514986)
```

Comment:
Due to the change in the default choice of agency, the track positions changed slightly. With the explicit choice of agencies (`provider=["usa", "bom"]`), I want to prevent that the values become invalid again if we change the default behavior again in the future.